.circleci/config.yml

```diff
@@ -66,6 +66,37 @@ jobs:
           path: packages/ml-sdk/test-results
       - store_test_results:
           path: packages/platform-sdk/test-results
+  integration_test:
+    parameters:
+      os:
+        type: executor
+      node-version:
+        type: string
+    executor: << parameters.os >>
+    working_directory: *working_directory
+    steps:
+      - checkout
+      - restore_cache: *restore_yarn_cache
+      - run:
+          name: Installing dependencies
+          command: yarn install --frozen-lockfile
+      - save_cache: *save_yarn_cache
+      - run:
+          name: Use environment variables
+          command: |
+            echo $IT_CTP_PROJECT_KEY
+            echo $IT_CTP_CLIENT_SECRET
+            echo $IT_CTP_CLIENT_ID
```

Comment:
I don't see the need for this step, I believe we can remove it now.

Reply:
it's to pass the variable for the integration tests otherwise it doesn't catch them

Reply:
and it runs everything in a specific environment which is what we wanted to do